src/CommitmentsV2/SFA.DAS.CommitmentsV2/Models/DraftApprenticeship.cs

```diff
@@ -108,6 +108,14 @@ public void Merge(DraftApprenticeshipDetails source, Party modifyingParty)
                 FlexibleEmployment.EmploymentEndDate = null;
             }
 
+            RecognisePriorLearning = source.RecognisePriorLearning;
+            if(RecognisePriorLearning == true)
+            {
+                PriorLearning ??= new ApprenticeshipPriorLearning();
+                PriorLearning.DurationReducedBy = source.DurationReducedBy;
+                PriorLearning.PriceReducedBy = source.PriceReducedBy;
+            }
+            
```

Comment:
I haven't tested this yet, but looking at where this code is happening, it I think it will cause problems on the routes when editing an apprenticeship. 

I think it may effectively reset the RPL data each time we edit a draft apprenticeship. 

Could we follow the pattern of adding the RPL data after the merge? Would that work?